src/components/test/AnnotationTimestamps-test.js

```diff
@@ -70,6 +70,23 @@ describe('AnnotationTimestamps', () => {
     assert.include(editedTimestamp.text(), '(edited another fuzzy string)');
   });
 
+  [
+    { withEditedTimestamp: true, expectedVariant: 'subtle' },
+    { withEditedTimestamp: 'subtle', expectedVariant: 'subtle' },
+    { withEditedTimestamp: 'prominent', expectedVariant: 'prominent' },
+  ].forEach(({ withEditedTimestamp, expectedVariant }) => {
+    it('renders edited timestamp with expected variant', () => {
+      fakeFormatRelativeDate.onCall(1).returns('another fuzzy string');
+
+      const wrapper = createComponent({ withEditedTimestamp });
+
+      const editedTimestamp = wrapper
+        .find('[data-testid="timestamp-edited"]')
+        .getDOMNode();
+      assert.equal(editedTimestamp.dataset.variant, expectedVariant);
```

Comment:
I decided to just test with a `data-variant` attribute, rather than testing the actual classes being added for every variant, to reduce coupling the test with implementation details.